android/build.gradle

```diff
@@ -79,7 +79,8 @@ android {
 
     defaultConfig {
         versionName project.ext.versionName
-
+        renderscriptTargetApi 16
```

Comment:
I think this needs `android:minSdkVersion="17"` right? This will need a final decision from @gubatron